lib/net/http/persistent.rb

```diff
@@ -1111,7 +1111,7 @@ def verify_mode= verify_mode
   end
 
   ##
-  # Sets the HTTPS verify_hostname.  Defaults to false.
+  # Sets the HTTPS verify_hostname.
```

Comment:
As per https://github.com/drbrain/net-http-persistent/pull/120 

> net/http/persistent should be thin wrapper, 

This setting actually comes from SSLContext's default and it defaults to true. The old comment is at best misleading.

See:
- https://github.com/ruby/openssl/blob/ade50769dfac81c994a5c7fe64884faf7d75ae44/lib/openssl/ssl.rb#L26
- https://github.com/drbrain/net-http-persistent/pull/120